terraform/scripts/setup.sh.tpl

```diff
@@ -1,32 +1,50 @@
 #!/bin/bash
 
-control_node_name="$(curl -s http://metadata.google.internal/computeMetadata/v1/instance/name -H 'Metadata-Flavor: Google')"
```

Comment:
Any thoughts about a simple "set -e" to bail out on any error, just to make things a bit shorter?